tests/bgp/test_traffic_shift.py

```diff
@@ -63,13 +63,13 @@ def verify_all_routes_announce_to_neighs(dut_host, neigh_hosts, routes_dut, ip_v
     return True
 
 
-def test_TSA(duthosts, enum_rand_one_per_hwsku_frontend_hostname, ptfhost,
+def test_TSA(duthosts, rand_one_dut_front_end_hostname, ptfhost,
```

Comment:
I think its a good idea to validate TSA/B functionality atleast once on each hwsku. I would suggest not changing the testcases in this file (atleast basic tsa/b tests)

Reply:
Hi @tjchadaga, Ok I agree. Reverted changes on the basic traffic-shift tests.